tests/testthat/test-seq_linter.R

```diff
@@ -5,6 +5,32 @@ test_that("other : expressions are fine", {
   expect_lint("function(x) { 1:(length(x) || 1) }", NULL, linter)
 })
 
+test_that("seq_len(...) or seq_along(...) expressions are fine", {
+  linter <- seq_linter()
+
+  expect_lint("function(x) { seq_len(x) }", NULL, linter)
+  expect_lint("function(x) { seq_along(x) }", NULL, linter)
+
+  expect_lint("function(x) { seq(2, length(x)) }", NULL, linter)
+  expect_lint("function(x) { seq(length(x), 2) }", NULL, linter)
+})
+
+test_that("finds seq(...) expressions", {
+  linter <- seq_linter()
```

Comment:
could you add a test for seq(dim(x)[1]) which is how dim would usually be used? it's pretty rare (never seen it before myself), so if the current approach doesn't work, just file a follow-up issue.

but if the current xpath is good enough just add tests to prevent regression

Reply:
Oh, wait. This should actually produce a lint:

``` r
seq(dim(data.frame())[1])
#> [1] 1 0
```

<sup>Created on 2022-07-26 by the [reprex package](https://reprex.tidyverse.org) (v2.0.1)</sup>

Reply:
right... don't worry about it though. just please file a follow-up issue

Reply:
and plz revert the incorrect test :)

Reply:
Is it okay if I add the correct test but comment it out for now?

Reply:
Created issue to track this: https://github.com/r-lib/lintr/issues/1474

Reply:
yep that's fine, though we'll have to be wary of comment_linter

Reply:
see :)